go.mod

```diff
@@ -0,0 +1,46 @@
+module github.com/crossplane-contrib/provider-argocd
+
+go 1.16
+
+require (
+	github.com/argoproj/argo-cd/v2 v2.0.1
+	github.com/crossplane/crossplane-runtime v0.13.0
+	github.com/crossplane/crossplane-tools v0.0.0-20201201125637-9ddc70edfd0d
+	github.com/google/go-cmp v0.5.2
+	github.com/pkg/errors v0.9.1
+	golang.org/x/oauth2 v0.0.0-20200902213428-5d25da1a8d43 // indirect
+	golang.org/x/tools v0.0.0-20201208062317-e652b2f42cc7 // indirect
+	google.golang.org/grpc v1.33.1
+	gopkg.in/alecthomas/kingpin.v2 v2.2.6
+	k8s.io/api v0.20.4
+	k8s.io/apimachinery v0.20.4
+	sigs.k8s.io/controller-runtime v0.8.3
+	sigs.k8s.io/controller-tools v0.5.0
+)
+
+replace (
```

Comment:
the joys of depending on another project that depends on k8s.io packages :sweat_smile: